### DISCUSSION TITLE
To trigger crash with sprite id: -1

### DISCUSSION THREAD

**Discussion by MasterNama17083** [nesbox/TIC-80#1617] [Ideas]
Hello. I've got idea let's see how did crash TIC-80?
https://user-images.githubusercontent.com/80534869/136331051-310e86d7-5bcd-4aff-b6ad-244713fadba7.mp4

**Code:**
```lua
t=0
x=96
y=24

function TIC()

	if btn(0) then y=y-1 end
	if btn(1) then y=y+1 end
	if btn(2) then x=x-1 end
	if btn(3) then x=x+1 end
	-- to trigger crash
        if btnp(4) then spr(-1)end
 
	cls(13)
	spr(1+t%60//30*2,x,y,14,3,0,0,2,2)
	print("Press A to trigger Crash",54,84)
	t=t+1
end
```

How it works?

**Comment by eggbanana173** [comment]
is a bug? (perdón no hablo bien ingles)

**Comment by nesbox** [comment]
I created an issue here https://github.com/nesbox/TIC-80/issues/1656
Thank you.
